files/build_templates/sonic_debian_extension.j2

```diff
@@ -68,8 +68,10 @@ sudo bash -c "echo \"DOCKER_OPTS=\"--storage-driver=overlay2\"\" >> $FILESYSTEM_
 # Copy docker start script to be able to start docker in chroot
 sudo mkdir -p "$FILESYSTEM_ROOT/$DOCKER_CTL_DIR"
 sudo cp $DOCKER_SCRIPTS_DIR/docker "$FILESYSTEM_ROOT/$DOCKER_CTL_SCRIPT"
-if [ $MULTIARCH_QEMU_ENVIRON == y ]; then
+
+if [[ $MULTIARCH_QEMU_ENVIRON == y  || $CROSS_BUILD_ENVIRON == y ]]; then
     DOCKER_HOST="unix:///dockerfs/var/run/docker.sock"
+    SONIC_NATIVE_DOCKERD_FOR_DOCKERFS=" -H unix:///dockerfs/var/run/docker.sock "
```

Comment:
>SONIC_NATIVE_DOCKERD_FOR_DOCKERFS [](http://example.com/codeflow?start=4&length=33)

This is defined at Makefile.work L214, could you reuse?

Reply:
As far as I understand in Makefile.work this variable has different meaning. It is a list of commands to be executed. In sonic_debian_extension.j2 file it is docker daemon socket to connect to.